core/templates/pages/community-dashboard-page/community-dashboard-page.controller.ts

```diff
@@ -134,6 +134,8 @@ angular.module('oppia').directive('communityDashboardPage', [
             UserService.getUserInfoAsync().then(function(userInfo) {
               if (userInfo.isLoggedIn()) {
                 ctrl.username = userInfo.getUsername();
+              } else {
+                ctrl.username = 'Guest';
```

Comment:
This seems hacky. "null" would be more appropriate if there isn't a real user logged in.

Reply:
Done!